src/core/exec-invoke.c

```diff
@@ -2243,6 +2248,24 @@ static int bpffs_prepare(
                 if (fs_fd < 0)
                         _exit(EXIT_FAILURE);
 
+                bpf_delegate_to_string(delegate_cmds, number);
+
+                r = fsconfig(fs_fd, FSCONFIG_SET_STRING, "delegate_cmds", number, 0);
```

Comment:
`number` -> `BPF_DELEGATE_TO_STRING(delegate_cmds)`

Reply:
The kernel API accepts a numerical bitmask.
When you print the mount options the bitmask is pretty printed into a list of tokens, but when you set the options, the kernel accepts only a number.